### PR TITLE
Registy MVP Docs

### PR DESCRIPTION
**Pull request by ngrayluna**
## Description

W&B Registries MVP Docs

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [X] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [X] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/registry/configure_registry.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+:::
+
+## Registry roles
+
+A user can have different roles, and therefore access, for each registry in their organization. Organization administrators can set a given user's permission.
+
+The proceeding table lists the different roles a user can have and their permissions:
+
+
+| Role | Permissions |
```
Comment (ngrayluna) [inline docs/guides/registry/configure_registry.md:22]: @noaleetz Can you fill these in? :)

File: docs/guides/registry/create_collection.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+:::info
+Permitted artifact types for a collection are configured when a registry is created. This means that, depending on the registry where you create a collection, you may or may not be able to specify which types of artifacts can be added to that collection. For more information about artifact types, see [LINK].
+:::
+
+
+<Tabs
+  defaultValue="alltypes"
+  values={[
+    {label: 'Unrestricted collection types', value: 'alltypes'},
```
Comment (ngrayluna) [inline docs/guides/registry/create_collection.md:24]: @noaleetz Temporary name. Not sure how we want to officially call the two?

Comment (noaleetz) [reply docs/guides/registry/create_collection.md:24]: what do you mean by Unrestricted collection types? Is this for editing the accepted types in a registry?

Comment (ngrayluna) [reply docs/guides/registry/create_collection.md:24]: @noaleetz Great question. This is a placeholder. Unsure what to call regular registry where user can add any artifact type.

**Comment by ngrayluna** [comment]
Duplicate of: https://github.com/wandb/docodile/pull/689/files

File: docs/guides/registry/create_collection.md (hunk 2 above)
Comment (noaleetz) [reply docs/guides/registry/create_collection.md:24]: when you say regular registry you mean custom registry?

**Pull request closed without merging**